priiloader/Makefile

```diff
@@ -172,6 +172,9 @@ post_build:
 	@echo creating priiloader.app for installer
 	@echo
 	@./OpenDolBoot $(OUTPUT).dol ../Installer/data/priiloader.app
+# vwii needs the loader version, since BC-NAND is loaded at the same addr as priiloader
+	@echo copying priiloader_vwii.app for installer

```

Comment:
Would make this simpler, yeah.

Reply:
i'd change it, the loader makes it possible to load anything anywhere anyway

Reply:
Actually I'm not sure if this works, since the Wii versions needs the stub addded by OpenDolBoot, which isn't required by the HBC or vWii and might cause issues.

Reply:
from what i know, in vwii loader stubs @ 3400 are ignored and nandloaders jump right to the entrypoint. 
this is what fail0verflow did to make HBC work on vwii, they added their entrypoint to the dol header.
so i think it would still work?

Reply:
Ah you mean running the dol through `bootloader` and then the result through `OpenDolBoot` to add the stub should work for all usecases?
Will have to try that, and also hope it still works on a Wii heh

Reply:
ah yes, sorry for being unclear. i did mean build priiloader -> run it through bootloader (which uses the priiloader's loader to start it from another address) -> pass result through opendolboot
it should still work on the wii if the bootloader is not in the way of the stub